src/image-gallery/setup.py

```diff
@@ -44,7 +44,7 @@
     name='image_gallery',
     version=VERSION,
     description='Support for Azure Image Gallery',
-   # TODO: Update author and email, if applicable
+    # TODO: Update author and email, if applicable
```

Comment:
Why modify here?

Reply:
`azdev style` will prompt that it is not aligned

Reply:
OK